dashboard/config/linux/upstream-apparmor-kasan.config

```diff
@@ -8676,13 +8690,16 @@ CONFIG_HAVE_ARCH_KCSAN=y
 #
 CONFIG_NET_DEV_REFCNT_TRACKER=y
 CONFIG_NET_NS_REFCNT_TRACKER=y
+# CONFIG_DEBUG_NET is not set
```

Comment:
I think we want to enable this. Eric added it to do some additional debug checks in hot network code.

Reply:
Oh, nice. Let me add it.

Reply:
Interesting, if I add `- DEBUG_NET: [v5.18]` to bits/base.yaml (assuming that means "enable for all kernels after v5.18), then a number of configs starts failing for me:

```
upstream-kmsan-base failed:
base.yml:109: DEBUG_NET is not present in the final config
saved config before olddefconfig to dashboard/config/linux/upstream-kmsan-base.config.tmp
```

But at least KMSAN is synced to kmsan-v5.18-rc4, it doesn't even contain v5.18. How is this possible?

Reply:
It's actually v5.19.
But adding v5.19 won't work right now because we look at v5.19* tags to understand it's v5.19, but there were none yet (we will detect it's v5.19 when v5.19-rc1 is tagged).
Add `linux-next` tag to it instead of `v5.18`.

Or, any suggestions how to detect it's already a merge window for v5.19?

Reply:
Makefile still has:
```
VERSION = 5
PATCHLEVEL = 18
```

Reply:
Thanks, done.